src/bin/ayb.rs

```diff
@@ -169,10 +176,23 @@ async fn main() -> std::io::Result<()> {
             Err(err) => println!("Error: {}", err),
         }
     } else if let Some(matches) = matches.subcommand_matches("client") {
+        let config_path = if let Some(path) = matches.get_one::<PathBuf>("config") {
+            path.clone()
+        } else {
+            home_dir()
+                .expect("can't determine home directory")
+                .join(".ayb.json")
```

Comment:
The standard (at least on Linux) is to create config files inside `$HOME/.config/$APP_NAME`.
In this case that would be `$HOME/.config/ayb/ayb.json`

Reply:
Thank you! I switched over to `directories`. Hurray for standardizing across operating systems! :)